projects/ui/src/lib/components/po-dynamic/po-dynamic-form/po-dynamic-form-field.interface.ts

```diff
@@ -63,22 +84,31 @@ export interface PoDynamicFormField extends PoDynamicField {
    */
   searchService?: string | PoLookupFilter;
 
-  /** Máscara para o campo. */
+  /**
+   * Máscara para o campo.
+   *
+   * **Componentes compatíveis:** `po-input`, `po-number`, `po-decimal`, `po-password`
```

Comment:
pela regra do componente quando passada a propriedade mask ele renderiza um po-input

```suggestion
   * **Componentes compatíveis:** `po-input`, `po-password`
```

Reply:
Resolvido.